ChangeLog.md

```diff
@@ -3,6 +3,309 @@
 Notable changes to the FreeOrion project will be documented in this file.
 
 
+## [v0.4.6] - 2016-09-10
+
+
+### Key Changes
+
+- Reworked supply mechanics
```

Comment:
I'd missed it reading through, I think perhaps a (details below) bracket or movign it to the top of its section as Geoff suggests would also be useful)